core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog_json/CamelCatalogCacheManager.java

```diff
@@ -0,0 +1,241 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.fusesource.ide.camel.model.service.core.catalog_json;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.fusesource.ide.camel.model.service.core.catalog_json.components.Component;
+import org.fusesource.ide.camel.model.service.core.catalog_json.dataformats.DataFormat;
+import org.fusesource.ide.camel.model.service.core.catalog_json.eips.Eip;
+import org.fusesource.ide.camel.model.service.core.catalog_json.languages.Language;
+
+/**
+ * @author lhein
+ */
+public class CamelCatalogCacheManager {
```

Comment:
What do you think to use a singleton instead of defining static methods?
it will allow to more easily test in the future by being able to mock a CatalogCache